app/models/question_type.rb

```diff
@@ -39,42 +45,47 @@ def human_name
 
   # returns whether this is a numeric type
   def numeric?
-    properties.include?("numeric")
+    properties.include?('numeric')
   end
 
   # returns whether this is an SMSable type
   def smsable?
-    properties.include?("smsable")
+    properties.include?('smsable')
   end
 
   # returns whether this question type makes sense to be printable on a form
   def form_printable?
-    properties.include?("form_printable")
+    properties.include?('form_printable')
   end
   alias_method :printable?, :form_printable?
 
   # returns whether this question type has options
   def has_options?
-    properties.include?("has_options")
+    properties.include?('has_options')
   end
 
   # returns whether this type has a timezone
   def has_timezone?
-    properties.include?("has_timezone")
+    properties.include?('has_timezone')
   end
 
   # returns whether this type is temporal
   def temporal?
-    properties.include?("temporal")
+    properties.include?('temporal')
```

Comment:
I just remembered that our styleguide says to always use double quoted strings. Can you fix your linter to adhere to that?